src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -791,6 +794,15 @@ def _apply_default_value_for_input(inputs: Dict[str, FlowInputDefinition], line_
                 updated_inputs[key] = value.default
         return updated_inputs
 
+    @staticmethod
+    def _get_node_referenced_flow_inputs(
+            node, flow_inputs: Dict[str, FlowInputDefinition]) -> Dict[str, FlowInputDefinition]:
+        node_referenced_flow_inputs = {}
+        for _, value in node.inputs.items():
+            if value.value_type == InputValueType.FLOW_INPUT and value.value in flow_inputs:
```

Comment:
If there is value in `node.inputs`, and its type is `InputValueType.FLOW_INPUT`, but not exist in `flow_inputs`.
Maybe we should throw exception?
Doesn't look like a valid yaml for me.